docs/source/Installation.rst

```diff
@@ -0,0 +1,62 @@
+Installation
+============
+
+SyReC is mainly developed as a C++ library.
+In order to make the tool as accessible as possible, it comes with an easy-to-use Python interface.
+
+We encourage installing SyReC via pip (preferably in a `virtual environment <https://docs.python.org/3/library/venv.html>`_):
+
+    .. code-block:: console
+
+        (venv) $ pip install mqt.syrec
+
+Once installed, start the SyReC editor GUI by running:
+
+    .. code-block:: console
+
+        (venv) $ syrec-editor
+
+In most practical cases (under 64-bit Linux, MacOS incl. Apple Silicon), this requires no compilation and merely downloads and installs a platform-specific pre-built wheel.
+
+.. warning::
+        Currently the FLINT2 dependency is not automatically installed. Thus, windows users need to install FLINT2 by hand.
+        Either by building `from source <http://www.flintlib.org/doc/building.html>`_ or downloading the corresponding `.zip file <http://www.flintlib.org/downloads.html>`_.
+        We are working towards fixing this issue.
```

Comment:
Please, never blindly copy and paste code. This does not belong here.

Reply:
done 👍🏻